styles/global.css

```diff
@@ -11,13 +11,6 @@ html {
   font-weight: 400;
   src: url(/fonts/DM-mono-v10.ttf) format('truetype');
 }
-@font-face {
-  font-display: swap;
-  font-family: 'DM Sans';
-  font-style: normal;
-  font-weight: 400;
-  src: url(/fonts/DM-sans-v11.ttf) format('truetype');
```

Comment:
I think you can also delete the `/public/fonts/DM-sans-v11.ttf` file

Reply:
I prefer to keep it in case we need it in the future. With this change it does not download to the client side so it's good enough.

Reply:
Is it not possible to revert it at all time via git if needed?